cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -321,13 +321,6 @@ describe('Data Table', () => {
   });
 
   describe('Truncation', () => {
-    it(`Should have head cells with class .${DATA_TABLE_CLASSES.TRUNCATED}`, () => {
-      cy.get(
-        `[data-cy='data-table-truncation'] .${DATA_TABLE_CLASSES.HEAD} .${DATA_TABLE_CLASSES.CELL}`
-      ).as('cells');
-      hasClassAssertion(`@cells`, DATA_TABLE_CLASSES.TRUNCATED);
```

Comment:
The test case has been removed as the class `-truncated` is no longer provided on Data Table head cells